drivers/sensor/apds9960/apds9960.c

```diff
@@ -458,11 +458,11 @@ static int apds9960_init(struct device *dev)
 	struct apds9960_data *data = dev->driver_data;
 	int als_gain = 0;
 
-	data->i2c = device_get_binding(CONFIG_APDS9960_I2C_DEV_NAME);
+	data->i2c = device_get_binding(CONFIG_I2C_SS_0_NAME);
```

Comment:
this does not look right, this is a generic sensor that should work with any board or configuration, yet you make it depend on I2C_SS, i.e. arduino_101 and quark se. What we had before is a actually correct.

Reply:
ohh yeah I got you